src/logEntryProcessing.ts

```diff
@@ -1,5 +1,139 @@
+import { CloudWatchLogsLogEvent } from 'aws-lambda';
+
 export function isRequestLogEntry(line: string): boolean {
     return line.startsWith('START RequestId: ') ||
         line.startsWith('END RequestId: ') ||
         line.startsWith('REPORT RequestId: ')
+}
+
+export function fieldValue(text: string, fieldNameText: string, valueLength?: number): string {
+    const colonAndSpaceSeparator = 2;
+    return text.substr(text.indexOf(fieldNameText)+fieldNameText.length + colonAndSpaceSeparator, valueLength).trim();
+}
+
+/**
+ * Parse an AWS lambda report field into a field name and a value
+ * @param rawField The raw field that looks something like "Field Name: value unit"
+ */
+export function parseReportField(rawField: string): [string, any] {
+    const [rawFieldName, rawFieldValue] = rawField.split(':').map(s => s.trim());
+
+    const fieldNameNoSpaces = rawFieldName.replace(/ /g, '');
+    const fieldName = fieldNameNoSpaces.charAt(0).toLowerCase() + fieldNameNoSpaces.slice(1);
+
+    const [value, unit] = rawFieldValue.split(' ');
+
+    if (unit == 'ms' || unit == 'MB') {
```

Comment:
Should this be [strict equality](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Equality_comparisons_and_sameness#strict_equality_using) (===)?

```suggestion
    if (unit === 'ms' || unit === 'MB') {
```